src/bootstrap/flags.rs

```diff
@@ -106,18 +106,18 @@ impl Flags {
 Usage: x.py <subcommand> [options] [<paths>...]
 
 Subcommands:
-    build       Compile either the compiler or libraries
-    check       Compile either the compiler or libraries, using cargo check
+    build, b    Compile either the compiler or libraries
```

Comment:
Doesn't `cargo` does not have the additional `, b`?

Reply:
No, it doesn't, but I think it should. I added it to promote discoverability.

Reply:
I think we can promote discoverability with `(b)uild` too.

Reply:
> No, it doesn't, but I think it should. I added it to promote discoverability.

Let me go add it.